src/motor.py

```diff
@@ -22,34 +22,58 @@ def out(self, pin, set_to):
         """Helper function to set 0 or 1 to gpio pin."""
         GPIO.output(self.pins[pin - 1], set_to)
 
-    def step_one(self):
+    def step_zero(self):
         """Setting gpio pins and sleeping."""
         self.out(1, 1)
         self.out(2, 0)
         self.out(3, 0)
         self.out(4, 0)
-        print('im at step 1')
+
+    def step_one(self):
+        """Setting gpio pins and sleeping."""
+        self.out(1, 1)
```

Comment:
We could consider a simplification here. Extend the self.out method to set all pins like so:
`self.out([1, 0],[2,1],[3,0],[4,0]`
Would make save quite a few LOC (lines of code)